usr/libexec/greenboot/greenboot.sh

```diff
@@ -70,7 +70,7 @@ case "$@" in
     run_red_scripts
     ;;
   *)
-    echo -e "\e[31mIllegal Command\e[0m" >&2
```

Comment:
were these color codes? Decided we didn't like them?

Reply:
They weren't picked up by the journal :( if there's a way to get them to work, I'd still like to have them very much